app/routes/task_routes.py

```diff
@@ -1 +1,104 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, abort, make_response, Response, request
+import requests
+from app.models.task import Task
+from datetime import datetime
+from sqlalchemy import asc, desc
+from .route_utilities import validate_model, create_model, get_models_with_filters
+from ..db import db
+import os
+
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+@tasks_bp.post("")
+def create_task():
+    request_body = request.get_json()
+    
+    if "title" not in request_body or "description" not in request_body:
+        return {"details": "Invalid data"}, 400
+    
+    task_data = {"title": request_body.get("title"), 
+                 "description": request_body.get("description"), 
+                 "completed_at": request_body.get("completed_at")}
```

Comment:
We don't necessarily need to do this part, we could pass in the request body since it has the `keys` and the accompanying `values` we need for our model construction.